tests/providers/ssh/hooks/test_ssh.py

```diff
@@ -742,7 +742,7 @@ def test_openssh_private_key(self):
 
     def test_oneline_key(self):
         with pytest.raises(Exception):
-            TEST_ONELINE_KEY = "-----BEGIN OPENSSH" + "PRIVATE KEY-----asdfg-----END OPENSSH PRIVATE KEY-----"
+            TEST_ONELINE_KEY = "-----BEGIN OPENSSHPRIVATE KEY-----asdfg-----END OPENSSHPRIVATE KEY-----"
```

Comment:
Mashed the name here so `detect-private-key` static check wouldn't fail. (Although this pre-commit hook didn't pickup what's in the `test_openssh_private_key` test in this file and maybe that's expected?)